### ISSUE TITLE
Debug crash from SendTransactionError

### ISSUE THREAD

**Issue by brittcyr** [Dual-Finance/risk-manager#154]
MNGO Gamma Ask Fill! size: 99250 market: MNGO/USDC price: 0.020242 makerId: 3350881801111 takerId: 0 time: 2023-02-03T16:17:46.792Z                                                                         │·······························
MNGO Cancelling OpenBook Orders 397020 MNGO @ 0.011901 6701763602220                                                                                                                                        │·······························
MNGO Cancelling OpenBook Orders 99250 MNGO @ 0.019836 3350881801110                                                                                                                                         │·······························
MNGO Cancelling OpenBook Orders 397020 MNGO @ 0.024999 6701763602221                                                                                                                                        │·······························
/home/ec2-user/risk-manager/node_modules/@solana/web3.js/src/connection.ts:5791                                                                                                                             │·······························
      throw new SendTransactionError(                                                                                                                                                                       │·······························
            ^                                                                                                                                                                                               │·······························
SendTransactionError: failed to send transaction: Transaction simulation failed: Blockhash not found                                                                                                        │·······························
    at Connection.sendEncodedTransaction (/home/ec2-user/risk-manager/node_modules/@solana/web3.js/src/connection.ts:5791:13)                                                                               │·······························
    at runMicrotasks (<anonymous>)                                                                                                                                                                          │·······························
    at processTicksAndRejections (node:internal/process/task_queues:96:5)                                                                                                                                   │·······························
    at async Connection.sendRawTransaction (/home/ec2-user/risk-manager/node_modules/@solana/web3.js/src/connection.ts:5750:20)                                                                             │·······························
    at async Connection.sendTransaction (/home/ec2-user/risk-manager/node_modules/@solana/web3.js/src/connection.ts:5738:12)                                                                                │·······························
    at async sendAndConfirmTransaction (/home/ec2-user/risk-manager/node_modules/@solana/web3.js/src/utils/send-and-confirm-transaction.ts:35:21) {                                                         │·······························
  logs: []                                                                                                                                                                                                  │·······························
}                                                                                                                                                                                                           │·······························
error Command failed with exit code 1.                                                                                                                                                                      │·······························
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.                                                                                                                        


did not catch the error and process crashed

**Comment by brittcyr** [comment]
another error
BONK Gamma Ask Fill! size: 9305000 market: BONK/USDC price: 0.000001046 makerId: 3350924944475 takerId: 0 time: 2023-02-03T22:21:05.393Z                                                                    │·······························
BONK Cancelling OpenBook Orders 9839000 BONK @ 9.95e-7 3350924944474                                                                                                                                        │·······························
BONK Cancelling OpenBook Orders 534000 BONK @ 0.000001046 3350924944475                                                                                                                                     │·······························
/home/ec2-user/risk-manager/node_modules/@solana/web3.js/src/utils/send-and-confirm-transaction.ts:92                                                                                                       │·······························
    throw new Error(                                                                                                                                                                                        │·······························
          ^                                                                                                                                                                                                 │·······························
Error: Transaction 2akm3soGMQsFiJ5r9ACo4riu4zpNzCDnjmWqFusLk2V2EHYiuqt41vt9A6upds3KVt8QzUxnEDepsox6CggUQdTj failed ({"err":{"InstructionError":[1,{"Custom":59}]}})                                         │·······························
    at sendAndConfirmTransaction (/home/ec2-user/risk-manager/node_modules/@solana/web3.js/src/utils/send-and-confirm-transaction.ts:92:11)                                                                 │·······························
    at processTicksAndRejections (node:internal/process/task_queues:96:5)                                                                                                                                   │·······························
error Command failed with exit code 1.

**Comment by brittcyr** [comment]
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/home/ec2-user/risk-manager/node_modules/node-fetch/lib/index.js:1505
                        reject(new FetchError(`request to ${request.url} failed, reason: ${err.message}`, 'system', err));
          ^
FetchError: request to ----------------------------------------- failed, reason: socket hang up
    at ClientRequest.<anonymous> (/home/ec2-user/risk-manager/node_modules/node-fetch/lib/index.js:1505:11)
    at ClientRequest.emit (node:events:513:28)
    at ClientRequest.emit (node:domain:489:12)
    at TLSSocket.socketOnEnd (node:_http_client:518:9)
    at TLSSocket.emit (node:events:525:35)
    at TLSSocket.emit (node:domain:489:12)
    at endReadableNT (node:internal/streams/readable:1358:12)
    at processTicksAndRejections (node:internal/process/task_queues:83:21) {
  type: 'system',
  errno: 'ECONNRESET',
  code: 'ECONNRESET'
}
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

**Comment by brittcyr** [comment]
fixed in #227
